### PR TITLE
Prevent domify from making unnecessary network requests

### PR DESCRIPTION
**Pull request by ramlmn**
`document.createRange().createContextualFragment` helps in creating a document fragment by parsing HTML but additionally loads resources linked from the HTML.

This is solved by using the HTML `template` tag.

### CODE REVIEW

**Comment by ramlmn** [comment]
And presumably solves possible issue mentioned in [this comment](https://github.com/sindresorhus/refined-github/pull/581#issuecomment-313711235).

File: src/libs/show-names.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const fetchName = async username => {
 
 export default async () => {
 	const myUsername = getUsername();
-	const cache = (await getCachedUsers())[storageKey] || {};
+	const cache = (await getCachedUsers())[storageKey];
```
Comment (bfred-it) [inline src/libs/show-names.js]: 👍

**Comment by ramlmn** [comment]
@bfred-it Can you review the changes?

**Comment by ramlmn** [comment]
``` js
export default html => {
	const template = document.createElement('template');
	template.innerHTML = html;

	const fragment = template.content;

	// If it's a single element, return just the element
	if (fragment.firstChild === fragment.lastChild) {
		return fragment.firstChild;
	}

	// Return fragment for querying
	return fragment; // <-- Should this be `fragment` or `template`?
};
```

@bfred-it Is this good enough?

**Comment by bfred-it** [comment]
Yep! That'd be good! The last line is `return fragment` like it originally was.

**Comment by ramlmn** [comment]
Fixed it!

**Comment by bfred-it** [comment]
Excellent! I tested this both places it will affect (show-names, showRecentlyPushedBranches) in the future (#586)

**Comment by ramlmn** [comment]
#591 solves [#581 (comment)](https://github.com/sindresorhus/refined-github/pull/581#issuecomment-313711235 "Use GitHub public api to fetch usernames")

**Pull request merged**